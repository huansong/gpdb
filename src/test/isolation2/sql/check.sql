select * from gp_toolkit.gp_check_orphaned_files;
