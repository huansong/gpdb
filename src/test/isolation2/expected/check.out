select * from gp_toolkit.gp_check_orphaned_files;
 gp_segment_id | tablespace | filename 
---------------+------------+----------
(0 rows)
